core/tx_pool.go

```diff
@@ -111,6 +111,7 @@ var (
 	invalidTxMeter     = metrics.NewRegisteredMeter("txpool/invalid", nil)
 	underpricedTxMeter = metrics.NewRegisteredMeter("txpool/underpriced", nil)
 	overflowedTxMeter  = metrics.NewRegisteredMeter("txpool/overflowed", nil)
+	throttleTxMeter    = metrics.NewRegisteredMeter("txpool/throttle", nil)
```

Comment:
We could add a meter for the reorgs, then we could see how wildly we're spinning.

Reply:
We can add a few more metrics there too: how long reorgs take (timer), how many drops in between reorgs (hist), + whatnot the author wants :D

Reply:
Fixed!